astropy/coordinates/tests/accuracy/test_altaz_icrs.py

```diff
@@ -55,7 +55,7 @@ def test_against_hor2eq():
     # radec_expected = ra=3.30875 deg, dec=15.183416666666666 deg
     # radec_actual = ra=3.3094193224314625 deg, dec=15.183757021354532 deg
     # distance = 2.6285 arcsec
-    assert distance < 5 * u.arcsec
+    assert distance < 10 * u.arcsec
```

Comment:
Actually this is still a problem.  @eteq - maybe you ran tests without `--remote-data` (which falls back to the IERS-B values)?

I think the source of difference is that with `IERS_Auto` you get the last available values for `UT1-UTC` instead of zero (for out-of-range `IERS-B`).  That gives a time diff of 0.6 sec, which can mean up to 0.6 * 15 arcsec.

Looking at the testing for AltAz, I wonder if the discrepancies are related to time diffs?

Anyway, I propose that changing to the original 5 arcsec tolerance to 10 arcsec should be OK now that we understand it.